### ISSUE TITLE
Crash on shutdown if TempFiles not used on Windows

### ISSUE THREAD

**Issue by czhower** [eugenekryukov/jskit#3]
JSK.API.pas lines 465 crashes here if temp files are not used. I changed it locally to this although there are probably better fixes but involve changing other and more lines of code:

  if (JSLib <> 0) and (TempFiles <> nil) and (TempFiles.Count > 0) then

**Comment by a07272** [comment]
i change the code like this, it work for me

procedure UnloadJavaScriptCore;
var
  I: Integer;
begin
  {$ifdef mswindows}
  if (JSLib <> 0) then
    FreeLibrary(JSLib);
  if (TempFiles <> nil) and (TempFiles.Count > 0) then
    TempFiles.DisposeOf;
  {$endif}
end;

**Comment by czhower** [comment]
Thanks. I've since had to move to Node for the project as there were some other issues (unrelated to your code, related to webkit), but I will likely need jskit again for a different project in the future.

**Comment by felipedaragon** [comment]
Yup! This fixes it, but why this has not been merged yet? It's been here since... 2018
